accord-core/src/main/java/accord/coordinate/Coordinate.java

```diff
@@ -34,19 +34,19 @@
 import accord.messages.PreAccept.PreAcceptReply;
 import com.google.common.base.Preconditions;
```

Comment:
We've migrated to `Invariants` mostly, to aid debugging (and also it's a better name)